.github/workflows/build-php-table-backend-utils.yml

```diff
@@ -54,7 +57,7 @@ env:
 
 jobs:
   build_image_php74_table_backend_utils:
-    if: ${{ inputs.hasCodeChanged || inputs.isTag }}
+    if: ${{ inputs.hasCodeChanged || inputs.isTag || (inputs.isRequiredRepoChanged && inputs.isTag)}}
```

Comment:
Buildit závislé budeme jen u tagu? Takže bez tagu nepoznáš, že jsi rozbil třeba IE libku? Není to škoda?

Reply:
Jo mas pravdu :)